packages/engine-core/src/data-proxy/utils/getClientVersion.ts

```diff
@@ -25,21 +24,16 @@ async function _getClientVersion(config: EngineConfig) {
   }
 
   // if it's an integration version, we resolve its data proxy
```

Comment:
```suggestion
  // if it's an integration or dev version
  // we need to resolve the closest version that is rolled out on Data Proxy infra
  // if not queries would fail because that version is not available (we are missing automated deployments of these on Data Proxy infra)
  // So we resolve with the closest previous version published on npm 
```